nservicebus/upgrades/release-policy.md

```diff
@@ -42,11 +42,16 @@ Some examples:
  * Version 4.6 was released on 2014-05-01 this means that its latest patch release will be supported till 2015-05-01. This means that there will be no fixes minor versions after 2015-05-01. It is required to update to at least a newer minor version that is still supported as this version will receive a patch release.
  * A newer patch release will automatically mean that the previous patch release will be obsolete. Particular will apply a bugfix on the latest patch release but will not officially release a patch for a obsolete patch release. In other words, version X.Y.3 will not be patched (to version X.Y.3.1) when version X.Y.4 is the latest patch. That would then be released as version X.Y.5
 
-It is strongly recommended to upgrade frequently enough to stay on a supported version. For the best upgrade experience, upgrade from one major version to the next major version. For example, when using NServiceBus Version 4.7.2 and intending to upgrade to NServiceBus Version 6.0.0, first upgrade to the latest NServiceBus Version 5.x release (e.g. Version 5.2.19) and follow the suggested API upgrade guides and [ObsoleteAttribute messages](https://msdn.microsoft.com/en-us/library/system.obsoleteattribute.aspx). Then repeat the exercise to upgrade from Version 5.x to Version 6.0.0.
+It is strongly recommended to upgrade frequently enough to stay on a supported version. For the best upgrade experience, upgrade from one major version to the next major version. For example, when using NServiceBus Version 4.7.2 and intending to upgrade to NServiceBus Version 6.0.0, first upgrade to the latest NServiceBus Version 5.x release (e.g. Version 5.2.19) and follow the suggested API upgrade guides and deprecation messages. Then repeat the exercise to upgrade from Version 5.x to Version 6.0.0.
 
 [Contact support](https://particular.net/support) if there are any bugfixes that needs to be back-ported to a specific version.
 
 
+### Deprecation
+
+Public API methods and properties are deprecated using the [ObsoleteAttribute messages](https://msdn.microsoft.com/en-us/library/system.obsoleteattribute.aspx). Continued usage of a deprecated method or property will result either in a compiler warning or error. A warning message indicates the version in which the deprecation will start to generate a compiler error (usually the next major version). The message will also indicate the version in which the deprecated method or property will be removed. Deprecations in minor releases will always generate a compiler warning and not an error.
```

Comment:
@adamralph  in an effort to keep things moving quickly in docs PRs are merged ASAP. If you would prefer this not to happen then you need to add a WIP.

Tweaking of PRs is something that happens in the docs repo regularly. For trivial things it greatly reduces the friction for PRs. 

Since we care less about history in docs, if ever something is not correctly fixed, then it can always be fixed after the fact. 

Also a significant number of small changes are made by the docs maintainers with no PR. So the change Indu did could have been done after the PR.

If you want to further tweak the md then submit a another PR and mark it as WIP so we can have an extended conversation about it